contents/docs/ai-engineering/_snippets/anthropic.mdx

```diff
@@ -44,6 +73,27 @@ response = client.messages.create(
 print(response.content[0].text)
 ```
 
+```typescript
+const response = await client.messages.create({
+  model: "claude-3-opus-20240229",
```

Comment:
Would be great to default the model to sonnet latest (most common)

I also think that anthropic requires maxTokens